docs/GettingStarted/configuration/ExporterWebhook.md

```diff
@@ -120,6 +120,8 @@ For the Header X-Pelorus-Event: `failure`
 
 You can easily send a POST request using [Curl](https://curl.se) directly from the shell. You can store the payload data in a file or pass it as an argument to [Curl](https://curl.se). Below is an example of sending several requests to cover the lifecycle of an application:
 
+  > **NOTE:** To align with the nature of Prometheus, it is required that the event being sent to the webhook exporter should not have occurred more than 30 minutes prior to the time of sending.
```

Comment:
I think this note is important, but maybe it is in the wrong place, reading the docs it feels like it applies to all payloads of webhook, and it is just `deploytime` payload, right?

maybe right below deploytime payload specification would be better

Reply:
Moved that note to the deploytime, also added one note to commit time that it should be sent prior to sending deploytime.